### PR TITLE
Raise an exception for missing int64 functions

### PR DESCRIPTION
**Pull request by tstenner**
See #9

### CODE REVIEW

**Comment by tstenner** [comment]
See also https://github.com/sccn/liblsl/pull/16

File: pylsl/pylsl.py [hunk 1]
```diff
@@ -1291,7 +1291,20 @@ def resolve_stream(*args):
 except:
     print("pylsl: ContinuousResolver not (fully) available in your liblsl "
           "version.")
-        
+
+# int64 support on windows and 32bit OSes will be properly implemented starting
+# with library version 1.14
+if bitness != 32 and os_name != 'Windows' or lib.lsl_library_version() > 113:
```
Comment (cboulay) [inline pylsl/pylsl.py]: Please remove the library version check until integer handling is actually merged into liblsl and released.

Comment (tstenner) [reply pylsl/pylsl.py]: Done.

**Comment by tstenner** [comment]
I've rebased the PR against master and removed the version check

**Pull request merged**